### PR TITLE
Adds popshare option

### PR DESCRIPTION
**Pull request by davidlvargas**

### CODE REVIEW

File: povcalnet.ado [hunk 1]
```diff
@@ -258,8 +259,26 @@ qui {
 		}
 	}
 	
-	*---------- Poverty line
-	if ("`povline'" == "") local povline = 1.9
+	*---------- Poverty line/population share
+	if ("`popshare'" != ""){
+		if ("`povline'" != ""){
+			oi disp as err "povline and popshare cannot be used at the same time"
```
Comment (randrescastaneda) [inline povcalnet.ado]: I Think this should be `noi` rather than `oi`

File: povcalnet.ado [hunk 2]
```diff
@@ -258,8 +259,26 @@ qui {
...
+		if ("`povline'" != ""){
+			oi disp as err "povline and popshare cannot be used at the same time"
+			error
+		}
+		else{
+			loc povline = ""
+			loc pcall = "popshare"
+		}
+	}
+	else if ("`povline'" == "") {
```
Comment (randrescastaneda) [inline povcalnet.ado]: This condition is not mutually exclusive with the first condition and with the else statement.  I think the problem is that these lines, 
```stata
if ("`popshare'" != ""){
		if ("`povline'" != ""){
```
Should be just one line

**Review by randrescastaneda — CHANGES_REQUESTED** [review]
HI David, 

I think the code looks really good. There is one important comment regarding a `noi` instead of a `oi`, but the other comment is just about logical coherence. The function works, but the conditions do not exclude each other. 

Thanks you

**Comment by randrescastaneda** [comment]
Hi Davdi, 

one more thing. It is still not working with several values in popshare. For instance, it is not working like this, 

```stata
povcalnet, popshare(.2 .5) clear
povcalnet, popshare(.2 .5) clear fillgaps
```

However, I am not worry about it. We could create a loop and make the query. It is not a big deal. 

Thanks.

File: povcalnet.ado (hunk 1 above)
Comment (davidlvargas) [reply povcalnet.ado]: Awful typo, I'll fix it right away

File: povcalnet.ado (hunk 2 above)
Comment (davidlvargas) [reply povcalnet.ado]: Agree, I'll adjust it

**Comment by davidlvargas** [comment]
Hi Andres,

All fixed and rebased to ease the merge. Let me know if something is missing.

Thanks !
David

**Comment by randrescastaneda** [comment]
Perfect. All changes have been added. 
Thank you so much!
Best,

**Pull request merged**